suricata-ocsf/package.yaml

```
@@ -46,12 +46,13 @@ pipelines:
         type_uid: 400106,
         // --- Context (required) ---
         metadata: {
-          log_name: "suricata.flow",
-          version: "v1.4.0",
+          log_name: "flow",
```

```suggestion
          log_name: event.event_type,
```

Also applies to all the others.